xml/entity-decl.ent

```diff
@@ -276,6 +276,8 @@
 
 <!ENTITY x86           "x86">
 <!ENTITY amd64         "AMD64">
+<!ENTITY arm           "Arm">
+<!ENTITY arm64         "&arm;&nbsp;&aarch64;">
```

Comment:
i know everything validates from the DocBook perspective, but where do we have the &aarch64; entity defined?